### PR TITLE
Make files flamegraph.pl can understand

### PR DESCRIPTION
**Pull request by umanwizard**
It's really annoying that:

1. We have to ask customers to send us a copy of the `materialized` binary they're using, so that we can symbolicate traces;
2. We have to write some complicated command to do so.

Now, just download the symbolicated profile, and then run `flamegraph.pl < ~/Downloads/mz.fg > out.svg` (where flamegraph.pl comes from [here](https://github.com/brendangregg/FlameGraph) .

---
This change is [Reviewable](https://reviewable.io/reviews/materializeinc/materialize/6253)

### CODE REVIEW

**Review by benesch — APPROVED** [review]

File: src/materialized/src/http/prof.rs [hunk 1]
```diff
@@ -259,6 +262,42 @@ mod enabled {
...
+                            }
+                            s.push_str(&*sym);
+                        }
+                    }
+                    writeln!(&mut s, " {}", stack.weight).unwrap();
+                }
+                Ok(Response::builder()
+                    .header(
+                        header::CONTENT_DISPOSITION,
+                        "attachement; filename=\"mz.fg\"",
```
Comment (quodlibetor) [inline src/materialized/src/http/prof.rs:296]: typo: `attachement` -> `attachment`. And, while you're changing this line, even though using raw strings is longer they might be easier to read? `r#"attachment; filename="mz.fg""#` , idk might be equivalent.

**Review by quodlibetor — APPROVED** [review]
🔥 📈

File: src/materialized/src/http/prof.rs (hunk 1 above)
Comment (umanwizard) [reply src/materialized/src/http/prof.rs:296]: whoops, I temporarily became French.

Comment (umanwizard) [reply src/materialized/src/http/prof.rs]: btw, I don't really think the `r#` version is easier to read -- personal taste, I suppose.

**Pull request merged**